### DISCUSSION TITLE
Latest Windows update breaks OpenShell

### DISCUSSION THREAD

**Discussion by randomgrace** [Open-Shell/Open-Shell-Menu#813] [General]
Just did a windows update to this week's updates.  Left clicking on the Start Menu icon in the task bar doesn't result in a Start Menu.  ShiftLeftClick does result in the normal Windows 'menu' (if you can call it a menu).  In my windows update history I have the following:

August 10, 2021—KB5005033 (OS Builds 19041.1165, 19042.1165, and 19043.1165)

**Comment by randomgrace** [comment]
BTW - I have installed the latest release 4.4.169.

**Reply by smayekawa** [reply]
Similar thing happened to me when I installed optional (KB5004296).
The pull-down menu (my task bar is located on the top side of the desktop) disappears very quickly and cannot be used at all.
I uninstalled it and then the menu worked all right, but when I installed KB5005033 it happened again.

Note that when the menu is located not at the top, it works all right.

**Reply by randomgrace** [reply]
Thanks for the reply.  I subsequently followed the instructions to disable the 'News and Interests' in the taskbar (described in issue #795, and rebooted.  I was then able to move the taskbar to the top.

**Reply by smayekawa** [reply]
Thank you very very much!  I did not know #795.  Tried and the solution works perfect.

**Reply by FranBunnyFFXII** [reply]
Just a bit of an FYI about the issue, at least for me after disabling news and Interests function, the startmenu flashes once as its opened regardless of position on the screen. I'm sure that there will be an issue that pops up with this later. I am wondering if there will be an update that fixes this issue.
Thank you for the thread guys, it fixed the issue for now.

**Reply by unmog** [reply]
Had the same issue, thanks to pointing me to the information to workaround it. I hope it gets actually fixed soon. <3
